custom/workflow-helper/postgres-helper/jmeter/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos:7.5.1804
```

Comment:
The base image [centos:7.5.1804](https://hub.docker.com/layers/centos/library/centos/centos7.5.1804/images/sha256-cf49811e3cdb94cbdfd645f3888d7add06a315449cf2c7ca7b81c312f1e46c63?context=explore) does not support multiarch platform it is only built for amd64 looks like so it should not work on arm64 I think. Can we check with any other centos that supports both platforms? like [centos7](https://hub.docker.com/layers/centos/library/centos/centos7/images/sha256-b98bfc4a7094a98ed52d49f9558bb15a4cee9854f182d30085d459d4a55d2e03?context=explore), [centos7.9.2009
](https://hub.docker.com/layers/centos/library/centos/centos7.9.2009/images/sha256-1cef7b00a46a2a90c62eef9a9707d01d19c2f5e253b7ccc574ccaa775063d7e0?context=explore) or [any centos 8](https://hub.docker.com/_/centos?tab=tags&page=1&ordering=last_updated&name=8)

Reply:
have updated centos8